packages/@aws-cdk/cdk/lib/context.ts

```diff
@@ -34,6 +34,24 @@ export class Context {
     return new AvailabilityZoneProvider(scope).availabilityZones;
   }
 
+  /**
+   * Set the availability zone to be used in the given scope
+   * Given a scope it will set the AZs list in the first stack scope in which the given scope is defined
+   * Note that supplying an unresolved value such us the return value of getAZ() will throw
```

Comment:
Provide a positive example on how to use `Fn.getAZs` here (`Fn.select(0, Fn.get...`)